playbooks/robusta_playbooks/pod_troubleshooting.py

```diff
@@ -256,6 +256,99 @@ def get_debugger_warnings(data):
         return None
     return message
 
+@action
+def debugger_stack_trace(event: PodEvent, params: DebuggerParams):
+    """
+    Prints a stack track of a python process and child threads
+
+    Create a finding with the stack trace results.
+    """
+    pod = event.get_pod()
+    if not pod:
+        logging.info(f"debugger_stack_trace - pod not found for event: {event}")
+        return
+
+    process_finder = ProcessFinder(pod, params, ProcessType.PYTHON)
+    if not params.pid and not process_finder.get_lowest_relevant_pid():
```

Comment:
Wouldn't it be better to call it only once? (process_finder.get_lowest_relevant_pid())
We call it again at line 275

Isn't it enough to just check if the result of process_finder.get_lowest_relevant_pid() is empty?

in addition, don't you want to stop the playbook if there's no pid? (like return)